ts/package.json

```diff
@@ -34,7 +34,6 @@
   },
   "devDependencies": {
     "@types/google-protobuf": "^3.2.5",
-    "@types/node": "^10.7.1",
```

Comment:
With the removal of the node-http transport from grpc-web-client - this is no longer required.